brave-spancollector-scribe/src/main/java/com/github/kristofa/brave/scribe/SpanProcessingThread.java

```diff
@@ -105,9 +108,10 @@ public Integer call() {
     private void log(final List<LogEntry> logEntries) {
         final long start = System.currentTimeMillis();
         final boolean success = log(clientProvider.getClient(), logEntries);
```

Comment:
if we were going to use System.currentTimeMillis() to calculate this (which would be more expensive and less precise than System.nanoTime()), we'd probably want to do so conditionally, as if there's a Noop notifier, we'd be creating overhead for no reason.